fv3core/stencils/updatedzc.py

```diff
@@ -220,44 +78,29 @@ def update_dz_c(
             xfx = p_weighted_average_bottom(ut, dp_ref)
             yfx = p_weighted_average_bottom(vt, dp_ref)
     with computation(PARALLEL), interval(...):
+        gz_x = gz
+        gz_x = corners.fill_corners_2cells_mult_x(gz_x, gz_x, 1.0, 1.0, 1.0, 1.0)
+        gz_y = gz_x
+        gz_y = corners.fill_corners_2cells_mult_y(gz_y, gz_y, 1.0, 1.0, 1.0, 1.0)
```

Comment:
Suggestion: `gz_x`, `gz_y`, `xfx`, `yfx` are all temporaries used only in the context of this function. Are there any more explicit names you can think of if you have context for what these mean? I'm sure you want to keep them short since the `gz` expression below is complex, but a few more characters might give a lot more meaning.

Reply:
Good point, these gz_variables were temporaries to carry the corners filled in the x and y directions, but we actually don't need both, we just need 1 gz_tmp because of the functional nature of gtscript functions. xfx and yfx are kind of odd names for cell averaged averages of u and v. I've renamed and reorganized.